app/src/renderer/scripts/wallet.js

```diff
@@ -173,9 +173,10 @@ export function createSignedTx(tx, signature) {
 }
 
 // the broadcast body consists of the signed tx and a return type
-export function createBroadcastBody(signedTx) {
+// returnType can be block (inclusion in block), async (right away), sync (after checkTx has passed)
+export function createBroadcastBody(signedTx, returnType = `block`) {
   return JSON.stringify({
     tx: signedTx,
```

Comment:
just little nicety: if you want to rename signedTx to tx we can simplify line `179`

Reply:
I thought that naming the argument `signedTx` is a good help for the developer to know that he needs to pass the signed tx in there. What do you think?

Reply:
That's true, this name is more informative!